pkg/operator/target_config_reconciler.go

```diff
@@ -82,12 +81,12 @@ func NewTargetConfigReconciler(
 }
 
 func (c TargetConfigReconciler) sync() error {
-	operatorConfig, err := c.operatorConfigClient.KubeSchedulers().Get("cluster", metav1.GetOptions{})
+	operatorSpec, _, _, err := c.operatorClient.GetStaticPodOperatorState()
```

Comment:
`operatorClient` client is initialized through:

```
operatorClient, _, _ := genericoperatorclient.NewStaticPodOperatorClient(ctx.KubeConfig, operatorv1.GroupVersion.WithResource("kubeschedulers"))
```

which under the hood creates an informer for `kubeschedulers` gvr. Also, `c.operatorClient.GetStaticPodOperatorState()` uses the informer and gets the same "kubescheduler" spec. So the change is equivalent in what resource gets returned. Though, using informers instead so the rate is decreased from `~1.42` to `~1.24`.